benches/tables/fst/general_category.rs

```diff
@@ -1,8 +1,8 @@
 // DO NOT EDIT THIS FILE. IT WAS AUTOMATICALLY GENERATED BY:
 //
-//  ucd-generate general-category /home/andrew/tmp/ucd/12.1.0/ --exclude unassigned --enum --fst-dir benches/tables/fst
+//  ucd-generate general-category ./ucd --exclude unassigned --enum --fst-dir benches/tables/fst
```

Comment:
Sorry, I know this is a nit, but could you change the directory name to `ucd-13.0.0` instead of just `ucd`? It would probably be best to include the Unicode version in a more formal way somewhere, but for now, I've been using the directory name here to indicate which Unicode version these files are generated for.

Reply:
No worries. I also agree including the version is worthwhile.